.github/workflows/linux_job.yml

```diff
@@ -215,6 +215,7 @@ jobs:
             echo "#!/usr/bin/env bash";
             echo "set -eou pipefail";
             # shellcheck disable=SC2016
+            echo 'export PATH=/opt/conda/bin:$PATH';
```

Comment:
This won't be needed after https://github.com/pytorch/pytorch/pull/139903 is merged